build.sbt

```diff
@@ -219,6 +219,7 @@ lazy val emberCore = libraryProject("ember-core")
     libraryDependencies ++= Seq(
       log4catsTesting % Test,
     ),
+    fork in Test := true,
```

Comment:
Hmm.  I thought this was already true everywhere.  Maybe it should be?  Would it make CI better or worse if it were? :thinking:

Reply:
Apparently it wasn't. I need to revert this, but I had enabled it to turn off CE tracing since it was incorrectly messing with stack traces (need to fix that)